docs/development.md

```diff
@@ -92,44 +95,49 @@ following chapters in a certain order:
     - spin up a bare redis container
 
       ```shell-session
-      docker run -d -p 6379:6379 --restart unless-stopped redis:latest
+      $ docker run -d -p 6379:6379 --restart unless-stopped redis:latest
       ```
 
-7.  Install the python dependencies by performing in the src/ directory.
+7.  Install the python dependencies by performing in the root directory:
 
     ```shell-session
-    pipenv install --dev
+    $ cd /path/to/paperless
+    $ pipenv install --dev
```

Comment:
Now that I'm looking at ordering, `pre-commit` won't be availible under the dependencies are installed and the `pipenv shell` is active, so step 4 needs to be after this